examples/negate.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+// How might I add my own support for --no-foo?
+
+// 1. const { parseArgs } = require('node:util'); // from node
+// 2. const { parseArgs } = require('@pkgjs/parseargs'); // from package
+const { parseArgs } = require('..'); // in repo
+
+const { values, tokens } = parseArgs({ strict: false, tokens: true });
+
+// Reprocess the option tokens and overwrite the returned values.
+// (NB: not supporting `multiples` in this code.)
+tokens
+  .filter((token) => token.kind === 'option')
+  .forEach((token) => {
```

Comment:
I find this to be a very elegant example of why having the detailed parse information is valuable :)